specification/servicenetworking/ServiceNetworking.Management/main.tsp

```diff
@@ -5,6 +5,7 @@ import "@azure-tools/typespec-azure-core";
 import "@azure-tools/typespec-azure-resource-manager";
 
 @armProviderNamespace
+@armCommonTypesVersion("v3")
```

Comment:
FYI. You can apply this at each version level so if you want to use v5 for latest common-type, you can.